fuzz/fuzz_helper/src/lib.rs

```diff
@@ -85,7 +85,7 @@ fn initialize(ctap: &mut Ctap<TestEnv>) -> ChannelID {
         cmd: CtapHidCommand::Init,
         payload: nonce,
     };
-    let mut assembler_reply = MessageAssembler::new();
+    let mut assembler_reply = MessageAssembler::default();
```

Comment:
fyi: Note that the idea of "new without default" is not to use "default", but to implement `Default` to let generic code use `T: Default` which is how they can call `::new()` because it's not bound to a trait. Non-generic code can just call `new()`. But given the change is done, no need to go back. Using `default()` is also fine.

Reply:
Ack, thanks.